### PR TITLE
Added azure release 13.1.0

### PR DESCRIPTION
**Pull request by whites11**
New azure 13.1.0 release

### CODE REVIEW

File: azure/v13.1.0/README.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+# :zap: Giant Swarm Release v13.1.0 for Azure :zap:
+
+This is next minor tenant cluster release for Node Pools Clusters.
+
+Release brings two new features that can be enabled by customers:
+
+- Auto Terminate Unhealth Nodes feature enables customers to discard unhealthy kubernetes nodes in favour of new ones added to the cluster. Follow [the guide](https://docs.giantswarm.io/basics/automatic-termination-of-bad-nodes/) to learn how to enable and disable this feature via annotations on Cluster CR.
```
Comment (MartaD) [inline azure/v13.1.0/README.md:7]:
```suggestion
- Automatic Termination Unhealth Nodes feature enables customers to discard unhealthy kubernetes nodes in favor of new ones added to the cluster. Follow [this guide](https://docs.giantswarm.io/basics/automatic-termination-of-bad-nodes/) to learn how to enable and disable this feature via annotations on Cluster CR.
```

File: azure/v13.1.0/README.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
+# :zap: Giant Swarm Release v13.1.0 for Azure :zap:
+
+This is next minor tenant cluster release for Node Pools Clusters.
```
Comment (MartaD) [inline azure/v13.1.0/README.md:3]:
```suggestion
This is the next minor workload cluster release for Node Pools Clusters.
```

File: azure/v13.1.0/README.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+
+## Change details
+
+
+### azure-operator [5.1.0](https://github.com/giantswarm/aws-operator/releases/tag/v5.1.0)
+
+#### Changed
+- Only submit Subnet ARM deployment when Subnet name or Subnet CIDR change.
+- Use controller-runtime instead of typed clients.
```
Comment (MartaD) [inline azure/v13.1.0/README.md:19]: could you add 'why' was this changed?

File: azure/v13.1.0/README.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+- Move provider-independent conditions implementation to external `giantswarm/conditions` and `giantswarm/conditions-handlers` modules.
+- Replaced Cluster `ProviderInfrastructureReady` with upstream `InfrastructureReady` condition.
+- Fix incorrect (too early) `Upgrading` condition transition from `True` to `False`.
+#### Added
+- Tenant cluster k8s client lookup is cached.
+- Add `terminate-unhealthy-node` feature to automaticaly terminate bad and unhealthy nodes in a Cluster.
```
Comment (MartaD) [inline azure/v13.1.0/README.md:25]:
```suggestion
- Add `terminate-unhealthy-node` feature to automatically terminate bad and unhealthy nodes in a Cluster, disabled by default.
```

**Review by MartaD — APPROVED** [review]

File: azure/v13.1.0/README.md (hunk 3 above)
Comment (whites11) [reply azure/v13.1.0/README.md:19]: uhm, they are kind of "internal optimizations" that have little to no impact for the customer. Should we remove them from here? They would be available in the component's changelog if the customer is interested.

Comment (MartaD) [reply azure/v13.1.0/README.md:19]: Let's talk on Monday 😁

Comment (whites11) [reply azure/v13.1.0/README.md:19]: Aaand we didn't talk about this :D

**Pull request merged**